department-of-veterans-affairs/caseflow
# Allow VSO users to page through their user queue

### Description
This PR allows VSO users to paginate through their user queue, as discussed in this [Slack thread](https://dsva.slack.com/archives/CJL810329/p1611255701008400)

### Testing Plan
1. Sign in as a VSO user.
2. Make sure multiple appeals are assigned to the user. Reassigning a `InformalHearingPresentationTask` from another VSO user is a good way to go. No need to assign more than 15 (see next step)
3. In `app/models/task_pager.rb`, change `TASKS_PER_PAGE` to 1.
4. Visit `/queue` and verify that paging works when running this branch (and doesn't work otherwise)

#### Not working:
<img width="999" src="https://user-images.githubusercontent.com/282869/105730989-eadc2e80-5efc-11eb-8eb6-bf0b50563b3c.png">

#### Working:
<img width="994" src="https://user-images.githubusercontent.com/282869/105731020-f4659680-5efc-11eb-8873-c2a80bbd0234.png">
